android/app/src/androidTest/java/org/ZingoLabs/Zingo/RustFFITest.kt

```diff
@@ -102,7 +102,7 @@ class ExecuteAddressesFromSeed {
         val mapper = jacksonObjectMapper()
 
         val server = "http://10.0.2.2:20000"
-        val chainhint = "main"
+        val chainhint = "regtest"
```

Comment:
these are set to main to return addresses etc. that look like real addresses. as they are not connected to any server (Offline tests)) i think it makes sense to return "real" outputs

Reply:
oh, yes, I don't know why I did this... sorry. I'll fix this.